source/leuven/wice_quick_start.rst

```diff
@@ -3,69 +3,103 @@
 
-If a GPU is necessary for the visualization process, it can be requested (max 1 GPU and max 8 cores for at most 16 hours). The available GPU is a single A100 which has been split in 7 GPU instances (one of which will be allocated to your job). Additionally, X11 forwarding should be enabled:
-
-::
+If a GPU is necessary for the visualization process, it can be requested (max 1 GPU instance 
+and total of 8 cores for at most 16 hours). 
+The available GPU is a single A100 which has been split in 7 GPU instances (one of which 
+will be allocated to your job). 
+Additionally, X11 forwarding should be enabled:::
 
    $ srun -N 1 -t 16:00:00 --ntasks-per-node=8 --gpus-per-node=1 -A lp_myproject -p interactive --cluster=wice --x11 --pty bash -l
 
 .. note::
 
-   The interactive partition is intended for relatively lightweight interactive work, such as compiling software, running small preprocessing scripts, small-scale debugging, or visualization. This is the reason why the amount of resources you can get in a job is limited on the interactive partition. In case you must do heavy computational work in an interactive way, it is also possible to submit interactive jobs to the other partitions. For instance suppose you need to debug a program using more than 8 cores. In that case you can use the command above to run an interactive job, changing the partition to ``batch``, ``gpu``, or ``bigmem`` and adapting the resources as needed.  Do note that in general it is recommended to run heavy computational work in a script which you run as a batch job (so without opening an interactive terminal on the compute node).
+   The interactive partition is intended for relatively lightweight interactive work, 
+   such as compiling software, running small preprocessing scripts, small-scale 
+   debugging, or visualization. 
+   This is the reason why the amount of resources you can get in a job is limited on the interactive partition. In case you must do heavy computational work in an interactive way, it is also possible to submit interactive jobs to the other partitions. For instance suppose you need to debug a program using more than 8 cores. In that case you can use the command above to run an interactive job, changing the partition to ``batch``, ``gpu``, or ``bigmem`` and adapting the resources as needed.  Do note that in general it is recommended to run heavy computational work in a script which you run as a batch job (so without opening an interactive terminal on the compute node).
+
+.. note::
+
+   We urge our users to recompile their software on wICE.
+   Using a software on wICE which is formerly compiled on Genius can lead to confusing
+   error messages, specifically in case of MPI applications.
+   This is true even for (mini)conda environments.
+   So, as a general rule of thumb, we propose that your software (including conda) be 
+   recompiled on wICE and put in a separate folder to distinguish the build from existing
+   software that suit Genius.
```

Comment:
Given that I branched off from the `development` branch, I did not have stuff from the `master` branch. 
I am also not gonna merge master into this branch, to avoid conflicts and wasted time to resolve all those conflicts.
I keep this comment open, and will cover this suggestion in another PR. This is already noted in the description of the current PR.

Reply:
Fair enough (though there is not much risk in locally git-pulling the master branch into your feature branch; if there are too many conflicts you can still abort it)